app/src/main/java/com/example/roomremix2/data/FloorPlanData.kt

```diff
@@ -19,14 +23,14 @@ class FloorPlanData {
         val items: List<FloorPlanItem>
     )
 
-    private fun saveFloorplanFile(floorPlan: FloorPlan, fileName: String) {
+    public fun saveFloorplanFile(floorPlan: FloorPlan, fileName: String) {
```

Comment:
Other way around.  Prefer private access modifiers, unless there's a good reason to give them more broad access.  More access == more security vulnerabilities.